### PR TITLE
fetch-in-react - Lindsay Jin

### PR DESCRIPTION
**Pull request by lindsay-jin**

### CODE REVIEW

File: fetch-in-react/my-app/src/UserDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  useEffect(() => {
+    async function loadUserDetails(userId: number) {
+      try {
+        const response = await fetch(
+          `https://jsonplaceholder.typicode.com/users/${userId}`,
+          {
+            method: 'get',
+          }
+        );
```
Comment (RobertGardner) [inline fetch-in-react/my-app/src/UserDetails.tsx:23]: Check response.ok here.

File: fetch-in-react/my-app/src/Users.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+  useEffect(() => {
+    async function loadUser() {
+      try {
+        const response = await fetch(
+          'https://jsonplaceholder.typicode.com/users',
+          {
+            method: 'get',
+          }
+        );
```
Comment (RobertGardner) [inline fetch-in-react/my-app/src/Users.tsx:29]: Here, too.

**Review by RobertGardner — CHANGES_REQUESTED** [review]
Nice, but get the error handling correct. See my Slack message.

File: fetch-in-react/my-app/src/Users.tsx (hunk 2 above)
Comment (lindsay-jin) [reply fetch-in-react/my-app/src/Users.tsx:29]: Changes have been updated. Thank you!

File: fetch-in-react/my-app/src/Users.tsx [hunk 3]
```diff
@@ -27,17 +27,21 @@ export function Users() {
...
+        }
         const result = await response.json();
         setUsers(result);
         setIsLoading(false);
         console.log('Success:', result);
       } catch (error) {
         setError(error);
         console.error('Error:', error);
+      } finally {
+        setIsLoading(false);
```
Comment (RobertGardner) [inline fetch-in-react/my-app/src/Users.tsx:41]: With this here, you can delete it on line 35.

**Review by RobertGardner — APPROVED** [review]

**Pull request merged**